libass/ass_utils.h

```diff
@@ -159,6 +163,17 @@ static inline uint32_t ass_bswap32(uint32_t x)
 #endif
 }
 
+static inline long ass_lrint(double x)
+{
+#if defined(_MSC_VER) && !defined(_M_ARM64EC) && (defined(_M_X64) || _M_IX86_FP == 2)
+    // Unlike GCC and Clang, Visual C++ can't inline lrint() without -fp:fast, so
+    // we provide our own implementation
+    return _mm_cvtsd_si32(_mm_set_sd(x));
+#else
+    return lrint(x);
+#endif
+}
```

Comment:
This adds a function call indirection on all platforms which we previously got rid off with `-fno-errno-math`. This is not acceptable. Instead `ass_lrint` should be a macro expanding to `lrint` on most platforms, or perhaps directly override `lrint` with a macro on MSVC

Reply:
It's a `static inline`, and no function call is emitted in practice (though I wouldn't object to `#define ass_lrint lrint` on non-MSVC): https://godbolt.org/z/Pcn7TMddo

Defining macros over standard library symbols already provided by the libc is generally UB.

Reply:
oh, it’s in the header not `.c`. Yeah, it’s unlikely to end up as a function call then